fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,204 @@
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Mobile No: ${volunteerInformation.mobileNo}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Preferred Hours: ${volunteerInformation.prefHours}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Experience Years: ${volunteerInformation.expYears}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text('Volunteer Possible Roles: ${volunteerInformation.possibleRoles.join(", ")}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Qualifications: ${volunteerInformation.qualifications.join(", ")}',
+                      style: TextStyle(
+                        fontSize: 14.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Availabilities:',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(
+                          'Monday: ${volunteerInformation.availabilities.Monday?.join(", ")}',
+                          style: TextStyle(
+                            fontSize: 18.0,
```

Comment:
Why are these the same font size as 'Volunteer Availabilities'? Make them smaller (using the theme) so there is a hierarchy of information